src/gc/config.d

```diff
@@ -26,7 +26,7 @@ struct Config
 {
     bool disable;            // start disabled
     ubyte profile;           // enable profiling with summary when terminating program
-    string gc = "conservative"; // select gc implementation conservative|manual
+    const(char)[] gc = "conservative"; // select gc implementation conservative|manual
```

Comment:
If we don't know all GCs that can be selected (b/c one w/ a new name can be linked in), then we can't have all names as string symbols. I'm now just taking a slice of the DRT-gcopt argument, which can come from the command line, a string constant in the binary, or if enabled from the environment (see [gc_config](https://dlang.org/spec/garbage.html#gc_config)).
We could malloc and copy the string to make it immutable, but at the time gc_init runs, no usercode has run, so it's hardly possible to change cmdline or env variables. And if someone does it, they simply have to deal w/ this.